webapp/store/search.js

```diff
@@ -69,7 +69,7 @@ export const actions = {
       .then(res => {
         commit('SET_QUICK_RESULTS', res.data.findPosts || [])
       })
-      .catch(() => {
+      .catch(_err => {
```

Comment:
Why this change?

Reply:
I think I added that to debug why the search feature wasn't working on @ogerly machine, but it was because the indices hadn't been set up... then maybe I forgot to remove and the linting added an underscore(?) 
anyways, it can be removed

Reply:
@ogerly used the `err` for logs and as he removed the logs I told him that it is nice to have possible parameters visible …